apps/core/src/pages/alternativeSignup/Otp.tsx

```diff
@@ -0,0 +1,144 @@
+import { useState } from "react";
+import {
+  Anchor,
+  Button,
+  createStyles,
+  Flex,
+  NumberInput,
+  Paper,
+  Text
+} from "@mantine/core";
+import { FormTitle } from "@zuri/ui";
+import { Link } from "react-router-dom";
+
+const useStyles = createStyles(theme => ({
+  form: {
+    maxWidth: "420px",
+    width: "100%",
+    // background: theme.colors.secondary[6],
+    padding: 40,
+    background: "white",
+    [`@media (max-width: ${theme.breakpoints.sm}px)`]: {
+      paddingLeft: 20,
+      paddingRight: 20
+    },
+    borderRadius: 15
+  },
+  label: {
+    fontSize: 16,
+    color: theme.colors.secondary[8]
+  },
+  outlineBtn: {
+    color: theme.colors.secondary[5],
+    borderColor: theme.colors.secondary[5]
+  },
+  link: {
+    color: theme.colors.secondary[9]
+  },
+  continueBtn: {
+    "&:invalid": { background: theme.colors.secondary[5] }
+  },
+  input: {
+    textAlign: "center",
+    borderColor: theme.colors.secondary[5],
+    borderWidth: 2,
+    fontSize: 20
+  }
+}));
+
+export const AlternativeSignupOtp = () => {
+  const { classes, theme } = useStyles();
+  return (
+    <Paper className={classes.form} radius={0}>
+      <FormTitle
+        title="Verify your email address!"
+        subtitle="We have sent your code "
+        boldSubtitle={"Kamsiyo@gmail.com"}
+        subtitleContinue=" Please enter it below before it expires."
+      />
+      <Text
+        color={theme.colors.tertiary[1]}
+        weight="700"
+        size={16}
+        mt={15}
+        mb={5}
+      >
+        Enter here
+      </Text>
+      <Flex gap={10} justify="space-between">
+        <NumberInput
+          hideControls
+          classNames={{ input: classes.input }}
+          h={45}
+          w={50}
+          maxLength={1}
+        />
+        <NumberInput
+          hideControls
+          classNames={{ input: classes.input }}
+          h={45}
+          w={50}
+          maxLength={1}
+        />
+        <NumberInput
+          hideControls
+          classNames={{ input: classes.input }}
+          h={45}
+          w={50}
+          maxLength={1}
+        />
+        <NumberInput
+          hideControls
+          classNames={{ input: classes.input }}
+          h={45}
+          w={50}
+          maxLength={1}
+        />
+        <NumberInput
+          hideControls
+          classNames={{ input: classes.input }}
+          h={45}
+          maxLength={1}
+          w={50}
+        />
+        <NumberInput
+          hideControls
+          classNames={{ input: classes.input }}
+          h={45}
+          maxLength={1}
+          w={50}
+        />
```

Comment:
This number input doesn't store the value anywhere.

Also Mantine provides a [pin input](https://mantine.dev/core/pin-input/) that you can use instead of recreating that design